sci-libs/opencascade/opencascade-7.8.1.ebuild

```diff
@@ -100,6 +107,8 @@ PATCHES=(
 	"${FILESDIR}/${PN}-7.7.0-jemalloc-lib-type.patch"
 	"${FILESDIR}/${PN}-7.8.0-cmake-min-version.patch"
 	"${FILESDIR}/${PN}-7.8.0-tests.patch"
+	"${FILESDIR}/${PN}-7.8.0-jemalloc-noexcept.patch"
+	"${FILESDIR}/${PN}-7.8.1-vtk_components.patch"
```

Comment:
Have these patches been sent upstream? Could you add a link here?

Reply:
Not yet, I'm still busy with fixing all the open bugs. So this fixes this for us, but I haven't had time to make a upstream PR out of this as that would require more checking and refactoring, especially for the vtk part.